js/adapt-languagePicker.js

```diff
@@ -38,7 +38,13 @@ define([
    * If it was, load it. If it wasn't, show the language picker
    */
   function onOfflineStorageReady() {
-    var storedLanguage = Adapt.offlineStorage.get('lang');
+    const urlParams = new URLSearchParams(window.location.search);
+
+    if (urlParams.has('_activeLanguage')) {
+      Adapt.offlineStorage.set('_activeLanguage', urlParams.get('_activeLanguage'));
+    }
```

Comment:
I kind-of envisaged this bit of code being in the xapi plugin rather than the languagePicker plugin

That said, it would be nice for review/development purposes to be able to stick the language in the URL and so not have to select every time you make a change to the code!

Either way would prefer the URL param to be `lang` rather than `_activeLanguage`

Reply:
I was trying to set _activeLanguage to remain consistent with the value used by xAPI. The '_lang' config setting for that is the xAPI verb language.
I will revert this to lang though if you believe the xAPI changes should be looking for `Adapt.offlineStorage.get('lang')` as well as `Adapt.offlineStorage.get('_activeLanguage')`

Reply:
oops sorry i was referring to the url param `?lang=en` seems more consistent with other webapps